packages/blockchain-wallet-v4-frontend/src/data/components/swap/utils.test.ts

```diff
@@ -1,15 +1,24 @@
-import { SwapProfile } from '@core/types'
-import { makeBaseAccount, makeCounterAccount } from 'data/components/swap/test-utils/makeAccount'
+import BigNumber from 'bignumber.js'
+
+import { SwapOrderDirection, SwapPaymentMethod } from '@core/network/api/swap/types'
+import { CoinType, SwapProfile } from '@core/types'
+import { makeAccount } from 'data/components/swap/test-utils/makeAccount'
+import { SwapBaseCounterTypes } from 'data/components/swap/types'
 import {
   getDirection,
   getPair,
   getPaymentMethod,
   getProfile,
-  getQuoteRefreshConfig
+  getQuoteRefreshConfig,
+  interpolatePrice,
+  isValidInputAmount
 } from 'data/components/swap/utils'
 
-const BASE = makeBaseAccount()
-const COUNTER = makeCounterAccount()
+// https://github.com/blockchain/service-trading-backend/blob/master/client-quote/src/test/kotlin/info/blockchain/albert/client/interpolate/LinearInterpolationTest.kt
+// assertEquals(7.5, interpolator.interpolate(50.0, 5.0, 100.0, 10.0, 75.0), 0.0)
```

Comment:
Can be removed?

Reply:
If that is for `interpolatePrice`, I would move it closer to the implementation of the util then.